lib/Retcon/Store/PostgreSQL.hs

```diff
@@ -148,31 +156,38 @@ instance Store PGStore where
       diffQ = "INSERT INTO retcon_diff (entity, id, is_conflict, content) VALUES (?, ?, ?, ?) RETURNING diff_id"
       opsQ = "INSERT INTO retcon_diff_conflicts (diff_id, content) VALUES (?, ?)"
 
-  resolveDiffs (PGStore conn _) did =
-      void $ execute conn sql (Only did)
+  resolveDiffs (PGStore conn _) did = do
+      execute conn "DELETE FROM retcon_diff_conflicts WHERE diff_id = ?" (Only did)
+      execute conn "UPDATE retcon_diff SET is_conflict = FALSE WHERE diff_id = ?" (Only did)
+      return ()
+
+  reduceDiff (PGStore conn _) patchID resolved = do
+      -- Change the corresponding conflicts
+      conops <- getOps
+      forM_ (map Only $ filterOps conops)
```

Comment:
does `executeMany conn "DELETE .... WHERE operation_id = ?" (map Only $ filterOps conops)` have the same semantics?